src/sentry/api/endpoints/project_dynamic_sampling.py

```diff
@@ -243,60 +301,127 @@ def get(self, request: Request, project) -> Response:
         query_time_range: QueryTimeRange = QueryTimeRange(time_now - stats_period, time_now)
 
         try:
-            root_transactions = self.__fetch_randomly_sampled_root_transactions(
+            # Fetch X random trace ids group by count_if(trace.parent_span, equals, "")
```

Comment:
This comment needs to be changed because it is not very accurate, we are not grouping by `count_if(trace.parent_span, equals, "")`